variables.tf

```diff
@@ -102,6 +102,12 @@ variable "enable_hive_metastore_metrics" {
   default     = false
 }
 
+variable "enable_hive_housekeeper" {
```

Comment:
```suggestion
variable "enable_hms_housekeeper" {
```

is that better ?

Reply:
README update - https://github.com/ExpediaGroup/apiary-data-lake/blob/master/VARIABLES.md